letsdrill.rb

```diff
@@ -1,12 +1,23 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
-
+  if (90..100).include?(integer)
```

Comment:
Interesting solution to use a range and `include?`!  You could also use Ruby's comparable methods since they are integers.